.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing to TAO
+
+When contributing to the TAO project, please first discuss the change you wish to make via issue.
+
+Contributions to the TAO codebase are done using the fork & pull model.
+This contribution model has contributors maintaining their own copy of the forked codebase (which can easily be synced with the main copy). The forked repository is then used to submit a request to the base repository to “pull” a set of changes. For more information on pull requests please refer to [GitHub Help](https://help.github.com/articles/about-pull-requests/).
+
+The TAO development team will review all issues and contributions submitted by the community of developers in the first in, first out order. During the review we might require clarifications from the contributor. If there is no response from the contributor within two weeks, the pull request will be closed.
+
+## Contribution process
+
+If you are a new GitHub user, we recommend that you create your own [free github account](https://github.com/signup/free). This will allow you to collaborate with the TAO development team, fork the TAO project and send pull requests.
+
+1. Check the open an closed issues for similar proposals of intended contribution before starting work on a new contribution.
+2. Create and test your work.
+3. Fork the TAO repository.
+4. Create a branch that follows the [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) branching model.
+5. Once development has been done, create a pull request that targets the development branch.
```

Comment:
A couple of days ago github added a new function to be able to create draft pull requests. Maybe we could highlight to create draft pull requests in case ...
- when PR is not in the final format
- when multiple extension is affected and the other PRs are not finished yet

Reply:
I agree that this makes sense for the second point, but a PR shouldn't be created if the development process is not yet done